src/app/core/campaign/campaign-store.service.ts

```diff
@@ -77,10 +74,12 @@ export class CampaignStoreService {
                   // accumulate values onto week
                   const weekTotals = acc.totals.groups[weekBeginString];
                   acc.totals.groups[weekBeginString] = {
+                    // allocated and availability will be null rather than undefined, null + 0 === 0
                     allocated: weekTotals.allocated ? weekTotals.allocated + day.allocated : day.allocated,
                     availability: weekTotals.availability ? weekTotals.availability + day.availability : day.availability,
                     allocationPreview: weekTotals.allocationPreview
-                      ? weekTotals.allocationPreview + day.allocationPreview
```

Comment:
addressed a bit of a bug here where we we're getting NaNs with summing undefined values